### PR TITLE
check internet connection

### PR DESCRIPTION
**Pull request by johnraftery**

### CODE REVIEW

File: src/pages/search/search.ts [hunk 1]
```diff
@@ -47,6 +49,16 @@ export class SearchPage {
...
+    if (!this.network.type || this.network.type == 'unknown' || this.network.type == 'none') {
+      let toast = this.toastCtrl.create({
+        message: 'Please enable a data connection to search',
+        duration: 2000,
+        position: 'bottom'
+      });
+      toast.present();
+      return
+    }
+
```
Comment (jamesdoc) [inline src/pages/search/search.ts]: Looks like this is repeated piece of code… can it be separated out?

Comment (johnraftery) [reply src/pages/search/search.ts]: It's very similar to one other place. I'm inclined to say it's not worth separating out. Do you feel strongly about it?

Comment (jamesdoc) [reply src/pages/search/search.ts]: Feel strong… no.

But… I see us using it more than just here.

Comment (johnraftery) [reply src/pages/search/search.ts]: How about I go ahead and merge this, and if/when we use this elsewhere we can be more clever about it?

**Pull request merged**